.github/workflows/pr_build.yaml

```diff
@@ -162,3 +162,25 @@ jobs:
         with:
           name: spiffe-helper
           path: spiffe-helper
+
+  build-images:
+    runs-on: ubuntu-22.04
+
+    permissions:
+      contents: read
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Install regctl
+        uses: regclient/actions/regctl-installer@main
+      - name: Build image
+        run: make docker-build
```

Comment:
Can we add a step here to save the images that are built, similar to this? https://github.com/spiffe/spire/blob/main/.github/workflows/pr_build.yaml#L186-L192

Reply:
That basically builds a tar of the same images uploaded to the registry. What is intent of the extra copy?

Maybe it should be removed from the spiffe/spire repo too.

A user can make the tarball for offline access themselves with docker, podman, skopeo, or crane pretty easily.

Reply:
valid point